onnxruntime/core/providers/cpu/activation/activations.h

```diff
@@ -89,9 +89,9 @@ struct Relu : public ElementWiseRangedTransform<T> {
   Status Init(const onnxruntime::NodeAttributes&) {
     return Status::OK();
   }
-  ElementWiseRangedTransform<T>* Copy() const {
+  ElementWiseRangedTransform<T>* Copy() const { // replace it with a macro. why this?
     using T1 = typename std::remove_pointer<decltype(this)>::type;
-    using T2 = typename std::remove_const<T1>::type;
+    using T2 = typename std::remove_const<T1>::type; //redundant?
```

Comment:
>//redundant? [](start = 53, length = 12)

should be remove_cv. Also do you need to remove reference along wit the ptr?

Reply:
You know, constructor can't be virtual. So here I was trying to find a way to bypass the restriction. 
Do you have a better solution for this?